packages/ui/src/services/global-config/types.ts

```diff
@@ -5,6 +5,7 @@ import type { IconConfig } from '../icon-config/types'
 export type GlobalConfig = {
   colors?: ColorConfig,
   icons?: IconConfig,
+  classHelpers?: ColorConfig,
```

Comment:
We already have ColorConfig here. 

Better have here config with class and properties - `helperConfigValues` and some settings, for example `classesPrefix`.

Make sure `helperConfigValues` will have a default value and simpler name 😁